Windows/MainWindow.xaml.cs

```diff
@@ -256,7 +348,7 @@ public partial class MainWindow {
 
             var loadData = targetFileType.GetMethod("LoadData", BindingFlags.Static | BindingFlags.Public | BindingFlags.FlattenHierarchy);
             Debug.Assert(loadData != null, nameof(loadData) + " != null");
-            fileData = (IMhwMultiStructFile) loadData.Invoke(null, new object[] {targetFile});
+            fileData = (IMhwMultiStructFile)loadData.Invoke(null, new object[] {targetFile});
```

Comment:
Please don't make pointless changes to my style like this.

Reply:
Sorry again, I'm not aware of this being a change...  Please ignore it and have it the way you like.